web/impact/impact/tests/test_analyze_judging_round_view.py

```diff
@@ -112,6 +115,8 @@ def test_get_with_program_criterion(self):
                                         options=[""])
         # This is where I gave up and just hard coded it!
         dists = {context.program.program_family.name: {0: 1, 1: 1}}
+        # clear cache since we added a new program family!
+        MatchingProgramCriterionHelper.program_families = None
```

Comment:
I see that you are having to clear the cache manually here in the test but how/where does this occur in the application? ( if we were to add a new program family in admin)

Reply:
That is not a scenario we would expect to encounter. Planning for it is serious YAGNI

Reply:
two questions:
- why do we expect to never add a program family?
- how would one clear the cache if we did change/add a program family?